easybuild/easyblocks/o/openfoam.py

```diff
@@ -483,6 +483,32 @@ def sanity_check_step(self):
             'dirs': dirs,
         }
 
+        # run motorBike tutorial case to ensure the installation is functional (if it's available);
+        # only for recent (>= v6.0) versions of openfoam.org variant
+        if self.looseversion >= LooseVersion('6') and self.looseversion < LooseVersion('100'):
+            openfoamdir_path = os.path.join(self.installdir, self.openfoamdir)
+            motorbike_path = os.path.join(openfoamdir_path, 'tutorials', 'incompressible', 'simpleFoam', 'motorBike')
+            if os.path.exists(motorbike_path):
+                cmds = [
+                    "cp -a %s %s" % (motorbike_path, self.builddir),
```

Comment:
As `self.build_in_installdir = True` is set we create this next to the installed software. Should we clean up afterwards / use a temporary path?
This probably depends on if we trust the check to error out in a sensible way if something goes wrong.

Reply:
fixed in fc37eb9